include/tins/sniffer.h

```diff
@@ -673,7 +673,7 @@ void Tins::BaseSniffer::sniff_loop(Functor function, uint32_t max_packets) {
     for(iterator it = begin(); it != end(); ++it) {
         try {
             // If the functor returns false, we're done
-            #if TINS_IS_CXX11 && !defined(_MSC_VER)
+			#if TINS_IS_CXX11 && defined(TINS_IS_CXX11_MSVCCOMPAT)
```

Comment:
Nice, this was added a while back when VC's support for C++11 was still not great.

Reply:
There's tabs here, could you use spaces?

Reply:
Done!